bundles/org.eclipse.e4.ui.progress/OSGI-INF/progress.xml

```diff
@@ -1,8 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<scr:component xmlns:scr="http://www.osgi.org/xmlns/scr/v1.1.0" name="org.eclipse.e4.ui.progress">
-   <implementation class="org.eclipse.e4.ui.progress.internal.ProgressServiceCreationFunction"/>
-   <service>
-      <provide interface="org.eclipse.e4.core.contexts.IContextFunction"/>
-   </service>
-   <property name="service.context.key" type="String" value="org.eclipse.e4.ui.progress.IProgressService"/>
```

Comment:
We might want to define a component property type for this `@ServiceContectKey(Class))` this will avoid needing to manually set such property